microservice-transaction-sample/docs/README.md

```diff
@@ -0,0 +1,443 @@
+# Microservice Transaction Sample
+
+This is a sample application for Microservice Transaction that uses Two-phase Commit Transactions in ScalarDB.
+You can find more information about Two-phase Commit Transactions in ScalarDB [here](https://github.com/scalar-labs/scalardb/tree/master/docs/two-phase-commit-transactions.md).
+
+## Prerequisites
+- Java (OpenJDK 8 or higher)
+- Gradle
+- Docker, Docker Compose
+
+## Sample application
+
+### Overview
+
+This is a simple EC site where you can order items and pay them with a credit card.
+
+There are two microservices called *Customer Service* and *Order Service* based on the [*Database-per-service* pattern](https://microservices.io/patterns/data/database-per-service.html) in this sample application.
+
+Customer Service manages customers' information including credit card information like a credit limit and a credit total.
+Order Service is responsible for order operations like placing an order and getting order histories.
+Each service has gRPC endpoints. Clients call the endpoints, and the services call the endpoints each other as well.
+Customer Service and Order Service uses MySQL and Cassandra through ScalarDB, respectively.
+
+![Overview](images/overview.png)
```

Comment:
Since this doc contains images, can we add the "microservice-transaction-sample/images" folder to the "microservice-transaction-sample/docs" folder as well?

Reply:
Done https://github.com/scalar-labs/scalardb-samples/pull/26/commits/e4f268f671db43bbe53f80a7926aca49b5129fab